src/common/components/PageHead/PageHead.tsx

```diff
@@ -32,6 +34,12 @@ export const PageHead = ({
       <meta content={pageDesc} name="description" />
       <meta content={pageName} name="og:title" />
       <meta content={pageDesc} name="og:description" />
+      <meta content={appName} property="og:site_name" />
+      <meta content="en" property="og:locale" />
+      <meta content="website" property="og:type" />
+      {/* Change this color to match the background of your app */}
+      {/* It'll appear in places like iOS notch area to make the app look better */}
+      <meta content="#000" name="theme-color" />
```

Comment:
haha, this happened to me in a project where I had to debug quite a bit 😄 . 
thanks for including this. 🙌

Reply:
> nice. in the future, we might be using [this](https://nextjs.org/blog/next-13-2#built-in-seo-support-with-new-metadata-api) 😄
> 
> if it is intuitive like ours. But currently, it might only support app dir. 🤔
> 
> ![CleanShot 2023-02-28 at 2 49 07](https://user-images.githubusercontent.com/53527664/221804726-ea875797-55de-494a-be4e-e607ce5ac448.png)

Yea defo. Once that's stable, we'll likely migrate over